standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -128,45 +130,86 @@ object StandardizationJob {
     spark
   }
 
-  private def readerFormatSpecific(dfReader: DataFrameReader, cmd: CmdConfig, dataset: Dataset): DataFrameReader = {
+  def getFormatSpecificReader(dfReader: DataFrameReader, cmd: CmdConfig, dataset: Dataset): DataFrameReader = {
```

Comment:
1. Please add docs for public methods
2. Why send in `dfReader` when we could create it as part of the method?

Reply:
Done (it does not show the code as `Outdated` because the documentation is added one line above)

Reply:
What about number 2?